src/lustre/lustreParser.mly

```diff
@@ -593,12 +595,39 @@ check:
     { A.AnnotProperty (mk_pos $startpos, name, e) }
 
 node_item:
+  | i = node_if_block { i }
+  | f = node_frame_block { f }
   | e = node_equation { A.Body e }
   | a = main_annot { a }
   | p = property { p }
   | p = check { p }
 
 
+
+node_if_block:
+  | IF; e = expr; 
+    THEN; 
+      l1 = list(node_item);
```

Comment:
Should we accept an empty list here? I would reject it. Same applies to the case without `else`.